manager/controllers/default/header.php

```diff
@@ -179,7 +179,7 @@ public function buildMenu($name, $placeholder)
                 $description = '<span class="description">'.$menu['description'].'</span>'."\n";
             }
 
-            $top = !empty($menu['children']) ? ' top' : '';
+            $top = ' top';
             $position = $idx <= 2 && $placeholder == 'navb' ? 'down' : 'up';
 
             $menuTpl = '<li id="limenu-'.$menu['id'].'"class="menu-'.$position.$top.'">'."\n";
```

Comment:
```suggestion
            $menuTpl = '<li id="limenu-' . $menu['id'] . '" class="menu-' . $position . ' top">' . "\n";
```

Reply:
Additionally, replace L192 with:
```php
$menuTpl .= '<a href="?a=' . $menu['action'] . $menu['params'] . '"' . $onclick . $title . $ariaLabel . '>' . $label . $description . '</a>' . "\n";
```

Then, finally, in the `_navbar.scss` file, change L111 from `.top:not(#modx-manager-search-icon) {` to:
```css
> li {
```